test/project_types/node/commands/generate/webhook_test.rb

```diff
@@ -9,31 +9,25 @@ class WebhookTest < MiniTest::Test
         def setup
           super
           ShopifyCli::ProjectType.load_type(:node)
-        end
-
-        def test_with_existing_param
           ShopifyCli::Tasks::Schema.expects(:call).returns(
             JSON.parse(File.read(File.join(ShopifyCli::ROOT, "test/fixtures/shopify_schema.json")))
           )
```

Comment:
The `test_with_selection` test is flakey but I'm not entirely sure what's causing it. I moved this expects to the setup since they were all calling it but that may or may not solve the flakiness.